satellite/console/service.go

```diff
@@ -1029,10 +1032,13 @@ func (s *Service) CreateProject(ctx context.Context, projectInfo ProjectInfo) (p
 
 		return nil
 	})
+		
 	if err != nil {
 		return nil, Error.Wrap(err)
 	}
 
+	s.analytics.TrackProjectCreated(auth.User.ID,currentProjectCount)
```

Comment:
pass in `currentProjectCount+1` and from inside `TrackProjectCreated`, don't add +1